actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -59,6 +59,7 @@ module AssetTagHelper
       #   when it is set to true.
       # * <tt>:nonce</tt>  - When set to true, adds an automatic nonce value if
       #   you have Content Security Policy enabled.
+      # * <tt>:defer</tt>  - When set to true, prevents adding preload links to header
```

Comment:
That's one of the things it does, but it's not really the main thing it does.  Maybe it is worth linking to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script or borrowing some of the explanation, eg

```suggestion
      # * <tt>:defer</tt>  -  When set to `true`, adds the `defer` HTML attribute, which indicates to the browser that the script is meant to be executed after the document has been parsed. Additionally, prevents sending the Preload Links header.
```

Reply:
Related: #46296

Reply:
I think this one is worth documenting because it does more than just add the HTML attribute. What do you think @jonathanhefner ?

Reply:
What about modifying the paragraph that discusses `Early Hints`?

```diff
diff --git a/actionview/lib/action_view/helpers/asset_tag_helper.rb b/actionview/lib/action_view/helpers/asset_tag_helper.rb
index eeba29e6c4..9d4bbb6859 100644
--- a/actionview/lib/action_view/helpers/asset_tag_helper.rb
+++ b/actionview/lib/action_view/helpers/asset_tag_helper.rb
@@ -41,8 +41,9 @@ module AssetTagHelper
       # When the Asset Pipeline is enabled, you can pass the name of your manifest as
       # source, and include other JavaScript or CoffeeScript files inside the manifest.
       #
-      # If the server supports Early Hints header links for these assets will be
-      # automatically pushed.
+      # If the server supports HTTP <tt>Early Hints</tt> and the +defer+ option
+      # is not enabled, Rails will push an <tt>Early Hints</tt> response that
+      # links to the assets.
       #
       # ==== Options
       #
```